From 243bf70e6cc757fb09cccbf2cfe601fe4e9bf211 Mon Sep 17 00:00:00 2001
From: "Simental Magana, Marcos" <marcos.simental.magana@intel.com>
Date: Tue, 8 Sep 2015 11:45:27 -0500
Subject: [PATCH 3/8] remove apparmor from unit file

Signed-off-by: Munoz, Obed N <obed.n.munoz@intel.com>
---
 daemon/libvirtd.service.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/daemon/libvirtd.service.in b/daemon/libvirtd.service.in
index c189f5e..5d64ae7 100644
--- a/daemon/libvirtd.service.in
+++ b/daemon/libvirtd.service.in
@@ -11,7 +11,6 @@ Before=libvirt-guests.service
 After=network.target
 After=dbus.service
 After=iscsid.service
-After=apparmor.service
 After=local-fs.target
 After=remote-fs.target
 Documentation=man:libvirtd(8)
-- 
2.15.1

