From 4bd6bbf1c1725d2342fcc56cd5f755b77979f812 Mon Sep 17 00:00:00 2001
From: "Simental Magana, Marcos" <marcos.simental.magana@intel.com>
Date: Wed, 25 Mar 2015 13:42:23 -0600
Subject: [PATCH 4/7] Use libsystemd instead of libsystemd-daemon

---
 m4/virt-systemd-daemon.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/m4/virt-systemd-daemon.m4 b/m4/virt-systemd-daemon.m4
index 8516e41..0e094c7 100644
--- a/m4/virt-systemd-daemon.m4
+++ b/m4/virt-systemd-daemon.m4
@@ -18,7 +18,7 @@ dnl <http://www.gnu.org/licenses/>.
 dnl
 
 AC_DEFUN([LIBVIRT_CHECK_SYSTEMD_DAEMON],[
-  LIBVIRT_CHECK_PKG([SYSTEMD_DAEMON], [libsystemd-daemon], [0.27.1])
+  LIBVIRT_CHECK_PKG([SYSTEMD_DAEMON], [libsystemd], [0.27.1])
 
     old_CFLAGS="$CFLAGS"
     old_LIBS="$LIBS"
-- 
2.5.0

