From ff6874a37289d3b9a6894420c9ae157529916fd9 Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Fri, 29 Jan 2016 09:48:04 -0600
Subject: [PATCH 5/8] drop timeout ping

Signed-off-by: Alberto Murillo <alberto.murillo.silva@intel.com>
Signed-off-by: Simental Magana, Marcos <marcos.simental.magana@intel.com>
---
 src/network/bridge_driver.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/network/bridge_driver.c b/src/network/bridge_driver.c
index 588b0d1..2f300f8 100644
--- a/src/network/bridge_driver.c
+++ b/src/network/bridge_driver.c
@@ -1589,6 +1589,7 @@ networkBuildDhcpDaemonCommandLine(virNetworkDriverStatePtr driver,
     cmd = virCommandNew(dnsmasqCapsGetBinaryPath(dnsmasq_caps));
     virCommandAddArgFormat(cmd, "--conf-file=%s", configfile);
     /* Libvirt gains full control of leases database */
+    virCommandAddArgFormat(cmd, "--no-ping");
     virCommandAddArgFormat(cmd, "--leasefile-ro");
     virCommandAddArgFormat(cmd, "--dhcp-script=%s", leaseshelper_path);
     virCommandAddEnvPair(cmd, "VIR_BRIDGE_NAME", def->bridge);
-- 
2.18.0

