From 4864e7ed75eeb127e9e6780b8dafd68bab578503 Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Fri, 29 Jan 2016 09:48:04 -0600
Subject: [PATCH 6/7] drop timeout ping

Signed-off-by: Alberto Murillo <alberto.murillo.silva@intel.com>
---
 src/network/bridge_driver.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/network/bridge_driver.c b/src/network/bridge_driver.c
index 362e294..4fcc2fa 100644
--- a/src/network/bridge_driver.c
+++ b/src/network/bridge_driver.c
@@ -1341,6 +1341,7 @@ networkBuildDhcpDaemonCommandLine(virNetworkDriverStatePtr driver,
     cmd = virCommandNew(dnsmasqCapsGetBinaryPath(dnsmasq_caps));
     virCommandAddArgFormat(cmd, "--conf-file=%s", configfile);
     /* Libvirt gains full control of leases database */
+    virCommandAddArgFormat(cmd, "--no-ping");
     virCommandAddArgFormat(cmd, "--leasefile-ro");
     virCommandAddArgFormat(cmd, "--dhcp-script=%s", leaseshelper_path);
     virCommandAddEnvPair(cmd, "VIR_BRIDGE_NAME", network->def->bridge);
-- 
2.5.0

