From 59456d8bc408289bcaa88fb04187e14c40026923 Mon Sep 17 00:00:00 2001
From: "Munoz, Obed N" <obed.n.munoz@intel.com>
Date: Tue, 8 Sep 2015 10:44:47 -0500
Subject: [PATCH 6/8] set default ciphers

Signed-off-by: Munoz, Obed N <obed.n.munoz@intel.com>
---
 m4/virt-tls-priority.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/m4/virt-tls-priority.m4 b/m4/virt-tls-priority.m4
index 44d9f68..db364f3 100644
--- a/m4/virt-tls-priority.m4
+++ b/m4/virt-tls-priority.m4
@@ -20,7 +20,7 @@ dnl
 AC_DEFUN([LIBVIRT_ARG_TLS_PRIORITY], [
   LIBVIRT_ARG_WITH([TLS_PRIORITY],
                    [set the default TLS session priority string],
-                   [NORMAL])
+                   [SECURE128])
 ])
 
 AC_DEFUN([LIBVIRT_CHECK_TLS_PRIORITY], [
-- 
2.15.1

