From 6bf90cfea85fade397c53fd71362446ec098cacb Mon Sep 17 00:00:00 2001
From: "Munoz, Obed N" <obed.n.munoz@intel.com>
Date: Tue, 8 Sep 2015 10:44:47 -0500
Subject: [PATCH 7/7] set default ciphers

Signed-off-by: Munoz, Obed N <obed.n.munoz@intel.com>
---
 src/rpc/virnettlscontext.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/rpc/virnettlscontext.c b/src/rpc/virnettlscontext.c
index 947038d..a0103ab 100644
--- a/src/rpc/virnettlscontext.c
+++ b/src/rpc/virnettlscontext.c
@@ -1227,7 +1227,7 @@ virNetTLSSessionPtr virNetTLSSessionNew(virNetTLSContextPtr ctxt,
     /* avoid calling all the priority functions, since the defaults
      * are adequate.
      */
-    if ((err = gnutls_set_default_priority(sess->session)) != 0) {
+    if ((err = gnutls_priority_set_direct(sess->session, "SECURE128",NULL)) != 0){
         virReportError(VIR_ERR_SYSTEM_ERROR,
                        _("Failed to set TLS session priority %s"),
                        gnutls_strerror(err));
-- 
2.5.0

